settings.gradle

```diff
@@ -63,6 +63,7 @@ includeWithFlags ':kafka',                               'java', 'publish', 'rel
 includeWithFlags ':kotlin',                              'java', 'publish', 'relocate', 'kotlin'
 includeWithFlags ':logback',                             'java', 'publish', 'relocate'
 includeWithFlags ':oauth2',                              'java', 'publish', 'relocate'
+includeWithFlags ':observation',                         'java', 'publish', 'relocate'
```

Comment:
Should we keep this as a separate module or make it part of the core? We already have Micrometer in our core, so I think we can move it into core if it doesn't pull in other dependencies such as Brave.

Reply:
Makes sense. We always release Micrometer Observation with Micrometer Core (that's the same project) and Core depends on Observation.

Reply:
Talked with the team and moving the implementation to the `:core` module